src/OrchardCore.Modules/OrchardCore.Secrets/Views/Admin/Index.cshtml

```diff
@@ -0,0 +1,177 @@
+@model SecretBindingIndexViewModel
+
+@{
+    int startIndex = (Model.Pager.Page - 1) * (Model.Pager.PageSize) + 1;
+    int endIndex = startIndex + Model.Entries.Count - 1;
+}
+
+<zone Name="Title"><h1>@RenderTitleSegments(T["Secrets"])</h1></zone>
+
+@* the form is necessary to generate an antiforgery token for the delete and toggle actions *@
+<form asp-action="Index" method="post" class="no-multisubmit">
```

Comment:
This page is very blue :). Let's use the same design as the other such UIs, like the content item list.

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/ebc31fa9-5e39-41a3-88fa-91227f1e5f80)

Reply:
It was not the case before, maybe after merging dev some styles has changed and some views need to be updated.

Reply:
Done